src/components/DropDown/DropDown.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
- No use for `React` it will be better to be removed.

Reply:
done